packages/web/utils/openseaHelpers.ts

```diff
@@ -0,0 +1,55 @@
+import { utils } from 'ethers';
```

Comment:
Arguable, but I think using the name better contextualizes the methods getting called on it.

```suggestion
import { utils as ethers } from 'ethers';
```